pkg/specgen/specgen.go

```diff
@@ -230,14 +230,6 @@ type ContainerSecurityConfig struct {
 	// If SELinux is enabled and this is not specified, a label will be
 	// automatically generated if not specified.
 	// Optional.
-	SelinuxProcessLabel string `json:"selinux_process_label,omitempty"`
```

Comment:
Is it necessary to completely remove this?

Reply:
It is not used.

Reply:
Why? The original intent of SpecGen was to make things as explicit as possible, removing parsing where possible - it seems like we're regressing on this here?

Reply:
We don't know at the time that the specgen is created what the labels are going to be.  The labels are creates in container/storage which happens after this is generate.

Reply:
These were originally intended for user-specified labels from `--security-opt label=` but it doesn't look like those compile cleanly into a complete process/mount label without outside intervention